drivers/spi/spi_ll_stm32.c

```diff
@@ -121,7 +121,12 @@ static int spi_stm32_dma_tx_load(const struct device *dev, const uint8_t *buf,
 		}
 	}
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+	/* use direct register location until the LL_SPI_DMA_GetTxRegAddr exists */
+	blk_cfg->dest_address = (uint32_t)&(cfg->spi->TXDR);
+#else
 	blk_cfg->dest_address = (uint32_t)LL_SPI_DMA_GetRegAddr(cfg->spi);
+#endif /* CONFIG_SOC_SERIES_STM32H7X */
```

Comment:
Similarly to what is done in the rest of the driver to ease readability let's use an abstraction function (for instance `ll_func_get_tx_reg_addr`).

Reply:
using` ll_func_get_reg_addr(spi, SPI_STM32_DMA_TX);`